app/controllers/user.js

```diff
@@ -54,3 +82,29 @@ exports.signUp = ({ user }, res, next) =>
         );
     })
     .catch(next);
+
+exports.usersPerPage = 2;
+
+exports.listUsers = (req, res, next) => {
+  try {
+    const decoded = jwt.decode(req.headers.token);
+  } catch (e) {
+    next(errors.badRequest(errorMsgs.invalidToken));
+    return;
+  }
+  const where = { limit: exports.usersPerPage, offset: 0 };
+  if (req.headers.page) where.offset = req.headers.page * exports.usersPerPage;
+
+  User.findAll(where)
+    .then(dbUsers => {
+      const toSendUsers = dbUsers.map(u => {
+        return { firstName: u.firstName, lastName: u.lastName, email: u.email };
```

Comment:
Remove brackets and return

Reply:
cant map directly to literal object notation because expects code within brackets.

Reply:
Test this .map(u => ({ new object }))